WebAutopark/Vehicle/VehicleType.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WebAutopark
```

Comment:
Wrong namespace. Move from Root ns to another.
U can create new class library project in solution and move all models and repos.
Like WebAutopark.DataAccess.

Reply:
![image](https://user-images.githubusercontent.com/79978383/118635618-72b3b480-b7dc-11eb-8885-c312ceac7d13.png)
Done